advanced_filters/admin.py

```diff
@@ -138,5 +139,23 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
                 return HttpResponseRedirect(url)
         return orig_response
 
+    def get_queryset(self, request):
+        # Filters by user unless the user is not a superuser or the settings specify not to
```

Comment:
Comment can be simplified to "Filters by user if not superuser or explicitly allowed in settings"